xarray/plot/plot.py

```diff
@@ -270,7 +278,11 @@ def line(darray, *args, **kwargs):
 
     # Rotate dates on xlabels
     if np.issubdtype(xplt.dtype, np.datetime64):
-        ax.get_figure().autofmt_xdate()
+        for xlabels in ax.get_xticklabels():
+            xlabels.set_rotation(30)
+            xlabels.set_ha('right')
```

Comment:
1. Let's add a link to [this StackOverflow post](https://stackoverflow.com/questions/17430105/autofmt-xdate-deletes-x-axis-labels-of-all-subplots) in a comment.
2. Do we even need this datetime64 logic with the latest version of matplotlib? matplotlib 2.2.x added some sort of built-in support for datetime64: https://github.com/matplotlib/matplotlib/pull/9779.

Reply:
Also, I am on 2.2.2...

Reply:
OK, thanks for checking!